### PR TITLE
minor fixes to grafana scripts

### PR DESCRIPTION
**Pull request by vimalk78**
Updated scripts to
  - use network instead of link
  - use podman or docker

### CODE REVIEW

**Review by opcm — CHANGES_REQUESTED** [review]
thanks for the change. Would you mind to make it run on docker-only based systems too as default? using podman should be an additional option. There are also README files in PCM which refer to docker (git grep docker)

**Comment by vimalk78** [comment]
> thanks for the change. Would you mind to make it run on docker-only based systems too as default? using podman should be an additional option. There are also README files in PCM which refer to docker (git grep docker)

@opcm updated as per your suggestion.

File: scripts/grafana/start.sh [hunk 1]
```diff
@@ -45,12 +46,14 @@ cp automatic_influxdb.yml provisioning/datasources/automatic.yml
...
 curl -o grafana_volume/dashboards/pcm-dashboard.json $1/dashboard
 
+echo "Creating influxdb network"
+${CTR_RUN} network create influxdb-network
 echo Starting influxdb
-docker run -d --name influxdb -p 8083:8083 -p 8086:8086 -v $PWD/influxdb_volume:/var/lib/influxdb influxdb:1.8.0-alpine
+${CTR_RUN} run -d --name influxdb -p 8083:8083 -p 8086:8086 -v $PWD/influxdb_volume:/var/lib/influxdb influxdb:1.8.0-alpine --network influxdb-network
 echo Starting telegraf
-docker run -d --name telegraf --link=influxdb -v $PWD/telegraf.conf:/etc/telegraf/telegraf.conf:ro telegraf
+${CTR_RUN} run -d --name telegraf --network=influxdb -v $PWD/telegraf.conf:/etc/telegraf/telegraf.conf:ro telegraf
```
Comment (opcm) [inline scripts/grafana/start.sh:54]: it should be influxdb-network

File: scripts/grafana/start.sh [hunk 2]
```diff
@@ -45,12 +46,14 @@ cp automatic_influxdb.yml provisioning/datasources/automatic.yml
...
 echo Starting grafana
-docker run -d --link=influxdb --name grafana -p 3000:3000 -v $PWD/provisioning:/etc/grafana/provisioning -v $PWD/grafana_volume:/var/lib/grafana grafana/grafana
+${CTR_RUN} run -d --network=influxdb --name grafana -p 3000:3000 -v $PWD/provisioning:/etc/grafana/provisioning -v $PWD/grafana_volume:/var/lib/grafana grafana/grafana
```
Comment (opcm) [inline scripts/grafana/start.sh:56]: same here

**Review by opcm — CHANGES_REQUESTED** [review]

File: scripts/grafana/start.sh (hunk 1 above)
Comment (vimalk78) [reply scripts/grafana/start.sh]: updated

File: scripts/grafana/start.sh (hunk 2 above)
Comment (vimalk78) [reply scripts/grafana/start.sh]: updated

**Comment by vimalk78** [comment]
Hello @opcm , can we get this merged?

**Review by opcm — APPROVED** [review]
LGTM. Sorry I thought this was merged already. Thanks for the reminder

**Pull request merged**